euphoria-core/src/main/java/cz/seznam/euphoria/core/client/util/Pair.java

```diff
@@ -19,12 +19,12 @@
 import java.util.Objects;
 
 /**
- * Pair of any types. The pair has to fit in memory and is read-only.
+ * A pair, i.e. a tuple of two elements.
```

Comment:
I see you removed the notice about mutability. What if we make the `Pair` really mutable by adding setters? I think most of implementations would benefit from this. Now we create a lot of short-term living objects because of `Pair` immutability. Is there any reason not go this way?
But maybe this a topic for separate issue.